.github/CODEOWNERS

```diff
@@ -1,7 +1,10 @@
 # Each line is a file pattern followed by one or more owners.
 
-# These owners will be the default owners for everything in
-# the repo. Unless a later match takes precedence,
-# these owners will be requested for review when someone
-# opens a pull request.
-*       @bjb28
+# These owners will be the default owners for everything in the
+# repo. Unless a later match takes precedence, these owners will be
+# requested for review when someone opens a pull request.
+* @ameliav @BenBreaksThings @nickviola @JCantu248 @dav3r @felddy @hillaryj @jsf9k @mcdonnnj
```

Comment:
```suggestion
* @ameliav @BenBreaksThings @nickviola @JCantu248 @dav3r @felddy @jsf9k @mcdonnnj
```

Reply:
This list of usernames should also be alphabetized.

Reply:
I've updated my suggestion to be alphabetized so this can be merged from the web interface if desired.

Reply:
Thanks all, I will be incorporating these changes but on a brand new branch. Dave suggested that this project use skeleton-docker instead of skeleton-generic so I'm going to pull from there instead, start fresh. I'm going to keep these branches as reference for now.

Reply:
> Thanks all, I will be incorporating these changes but on a brand new branch. Dave suggested that this project use skeleton-docker instead of skeleton-generic so I'm going to pull from there instead, start fresh. I'm going to keep these branches as reference for now.

@ameliav - I really think it will be easier for you to continue using this branch/PR to get up-to-date with `skeleton-generic` first, then follow the remaining steps mentioned in [this comment](https://github.com/cisagov/pca-report-library/pull/4#issuecomment-852437603).  If you follow the instructions listed at the top of this PR, you should be able to resolve all of the conflicts and get this PR ready for review.  Please let me know if you have any questions here.

Reply:
@dav3r Oh gotcha. I hadn't seen that comment- I'll follow those steps then and let you know if I have questions.

Reply:
Corrected as part of aeac8cefe17c8457b1e7892ed40ffe642549de89